marvin/report/observers/event_logger.py

```diff
@@ -56,17 +118,35 @@ def on_test_ended(self, event):
         test_script = event.test_script
         test_header = self._in_color('HEADER', 'TEST')
         status = self._colored_status(event.status)
+        self._current_status["test_status"] = event.status
+
+        self._suite_status.append(self._current_status)
+
+        if event.status != Status.PASS and len(event.exceptions) > 0:
```

Comment:
`len` is not needed.. just do `... and event.exceptions:` or if you think is clearer:  `... and any(event.exceptions):`